### ISSUE TITLE
Enhance docs to explain about English labels

### ISSUE THREAD

**Issue by johnbillion** [johnbillion/extended-cpts#89]
The documentation around the English language labels and how to provide translations is not clear. Needs a wiki page or something adding to the readme.

**Comment by huubl** [comment]
When modifying a build-in post type like page all the labels fallback to English. I found your comment below from 2018. So the only way is to add all the labels to `register_extended_post_type()`? Or are there other ways now?

https://github.com/johnbillion/extended-cpts/issues/66#issuecomment-447941601
> You need to provide and translate the complete labels array in full, just as you would when you register a post type with WordPress' built-in `register_post_type()` function. Unfortunately there's nothing that Extended CPTs can do about this. The automatic label generation only works for English labels.

**Comment by johnbillion** [comment]
Yes that's correct. You could probably call `get_post_type_labels( get_post_type_object( 'post' ) )` to get that list of labels so you don't need to redeclare them all.

(Extended CPTs was never originally designed for overriding existing post types, it's just something that does work.)

**Comment by huubl** [comment]
Thank you! Unfortunately `get_post_type_labels( get_post_type_object( 'post' ) )` returns the English labels.
